src/main/java/mate/academy/hibernate/relations/service/impl/ActorServiceImpl.java

```diff
@@ -1,16 +1,24 @@
 package mate.academy.hibernate.relations.service.impl;
 
+import java.util.Optional;
+import mate.academy.hibernate.relations.dao.ActorDao;
 import mate.academy.hibernate.relations.model.Actor;
 import mate.academy.hibernate.relations.service.ActorService;
 
 public class ActorServiceImpl implements ActorService {
+    private ActorDao actorDao;
+
+    public ActorServiceImpl(ActorDao actorDao) {
+        this.actorDao = actorDao;
+    }
+
     @Override
     public Actor add(Actor actor) {
-        return null;
+        return actorDao.add(actor);
     }
 
     @Override
-    public Actor get(Long id) {
-        return null;
+    public Optional<Actor> get(Long id) {
+        return actorDao.get(id).isPresent() ? actorDao.get(id) : Optional.empty();
```

Comment:
you can also do smth like that:
`return actorDao.get(id).orElseThrow();`
it can succinct your code

Reply:
the same to other impls